tests/cases/fourslash/signatureHelpCallExpressionJs.ts

```diff
@@ -0,0 +1,15 @@
+/// <reference path='fourslash.ts'/>
+
+// @checkJs: true
+// @allowJs: true
+
+// @Filename: main.js
+////function fnTest() { arguments; }
+////fnTest(/*1*/);

```

Comment:
can you add a call like `fnTest(1,2,3)` plus verify that semantic diagnostics count is 0?